index.html

```diff
@@ -65,164 +74,238 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
         </li>
       </ul>
       <a href="#" class="background">
-        <img src="background.png" alt="Background Image"/>
+        <img src="background.png" alt="Background Image" />
+      </a>
```

Comment:
- [ ]  I would suggest that if an image is not part of the content (aesthetic reasons) use CSS `background-image` instead of `img` tag (decorations are always background images, not img tags). [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.
- [ ] Also please make sure your desktop design doesn't effect your mobile version design as the aim is to learn to make a responsive website 

<img width="379" alt="Screenshot 2023-06-10 at 8 31 38 PM" src="https://github.com/bhushan354/Portfolio_Project/assets/44798044/96966ffb-4396-44bc-bfa2-d4002471ee7b">

Reply:
remove all unnecessary things - done , also thanks for such great review i have corrected all the errors